app/projects/models/base_models.py

```diff
@@ -704,11 +762,22 @@ class Bus(TopologyNode):
     input_ports = models.IntegerField(null=False, default=1)
     output_ports = models.IntegerField(null=False, default=1)
 
+    def to_datapackage(self):
+        dm = model_to_dict(self, fields=["type", "name"])
+        dm["facade"] = "bus"
+        dm["balanced"] = "True"
+        dm["excess"] = "False"

```

Comment:
Those are just suggestions and will change depending on the OpenPlan bus facade provided by @uvchik